### DISCUSSION TITLE
GET Request transformer returning blank after 3rd expansion

### DISCUSSION THREAD

**Discussion by datadumppro** [Budibase/budibase#7239] [Help]
Hello all, 

Is anyone able to tell me why this get request pull is returning a blank table upon the 3rd expansion using the transformer function? I am under the impression that the logic should not change considering the syntax but may be mistaken. Public XML file is available here: http://reports.ieso.ca/public/OntarioZonalDemand/PUB_OntarioZonalDemand.xml

Thanks in advance
![Original Pull](https://user-images.githubusercontent.com/58442501/184224890-f7633f34-ae71-4370-99b3-92585c9453b0.JPG)
![Failed Pull](https://user-images.githubusercontent.com/58442501/184224886-0e05c6c6-87a1-405a-a2a9-8e0d0a7808f4.JPG)

**Comment by melohagan** [comment]
Hi @datadumppro 

You need to do `data.DocBody.DeliveryDate`

Also *ZonalDemands* looks like a typo.
